Dockerfile

```diff
@@ -1,26 +1,33 @@
-FROM golang:1.14.2-buster
+FROM golang:1.14.2-alpine AS build
 
-# Install deps
-RUN apt-get update && apt-get install -y \
-  libssl-dev \
-  ca-certificates
+RUN apk add --no-cache openssl-dev git build-base
```

Comment:
Why was `ca-certificates` removed? Should it be in the run image?

Reply:
Very observant! 

As the package builds, it seems it is not (explicitly) required here. If it were, dependencies would simply not be fetched. (But it's not there 'cause I simply forgot). 

Looking through Alpine's package setup it seems that git requires curl which requires ca-certificates. So that explains why it works. Ref: https://pkgs.alpinelinux.org/package/v3.3/main/x86/git